docs/dev/clusteroperator.md

```diff
@@ -75,11 +75,35 @@ Here are the guarantees components can get when they follow the rules we define:
 
 There are a set of guarantees components are expected to honor in return:
 
-1. A component doesn't report the `Available` status condition the first time until they are completely rolled out (or within some reasonable percentage if the component must be installed to all nodes)
-2. A component reports `Failing` when it can't accomplish its task. This might be very broad - the API servers are down so I failed to update - or narrow - I couldn't update the last secret. In either case, Failing communicates something is wrong.
-3. A component reports `Progressing` when it is rolling out new code, propagating config changes, or otherwise moving from one steady state to another. It should not report progressing when it is reconciling a previously known state. If it is progressing to a new version, it should include the version in the message for the condition like "Moving to v1.0.1".
-4. A component reports `Upgradeable` as `false` when it wishes to prevent an upgrade for an admin-correctable condition. The component should include a message that describes what must be fixed.
-5. A component reports when it has rolled out the new version of its operands
+1. A operator doesn't report the `Available` status condition the first time
+   until they are completely rolled out (or within some reasonable percentage if
+   the component must be installed to all nodes)
+2. An operator reports `Degraded` when its current state does not match its
```

Comment:
> An operator reports `Degraded` when its current state does not match its
   desired state resulting in a lower quality of service over a period of time.

nit: `An operator reports `Degraded` when its current state does not match its
   desired state over a period of time resulting in a lower quality of service .` is much more clearer that operators mark degraded when they have been trying to achieve desired but haven't achieved it for a period of time....

Reply:
agreed.  updated phrasing to match.